lighthouse-core/audits/content-width.js

```diff
@@ -33,32 +43,26 @@ class ContentWidth extends Audit {
     const windowWidth = artifacts.ViewportDimensions.outerWidth;
     const widthsMatch = viewportWidth === windowWidth;
 
-    if (IsMobile) {
-      return {
-        score: Number(widthsMatch),
-        explanation: this.createExplanation(widthsMatch, artifacts.ViewportDimensions),
-      };
-    } else {
+    if (!IsMobile) {
```

Comment:
isMobile?

Reply:
I'm in the "leave better than you found it" camp. A local variable rename is low impact, and is unlikely to be done in a dedicated PR.

Reply:
I'm all for "leave better than you found it" when it comes to changes that specifically touch a file but a PR that's touching every single file in a category just might not be the absolute ideal place to try to fix all the variable names in all those files :) 

Now it turned out this was just the first file I happened to read and it's the only one this PR actually touches logic in, so it's all good, my comment can probably be ignored and I probably should've come back to erase it, but if the *all* the files had fixes like this, it would've made reviewing the i18n piece much harder that's all.

Reply:
I am just changing the control-flow since it had doubly nested if's, it was confusing looking when I removed the func to generate the explanations and moved it into the main func.  This is a no-op, just a control flow change. 😄